src/main/java/org/mozilla/zest/core/v1/ZestRequest.java

```diff
@@ -299,6 +300,14 @@ public void replaceTokens(ZestVariables tokens) {
 		this.setMethod(tokens.replaceInString(this.getMethod(), false));
 		this.setHeaders(tokens.replaceInString(this.getHeaders(), false));
 		this.setData(tokens.replaceInString(this.getData(), false));
+
+		List<ZestCookie> replacedCookies = new ArrayList<>();
+		for (ZestCookie cookie : this.cookies) {
+			replacedCookies.add(new ZestCookie(cookie.getDomain(), tokens.replaceInString(cookie.getName(), false),
```

Comment:
- How about replace the other fields (i.e. domain, path)? They can also have variables.
 - Does it really need to create new cookies? (It could use the setters.)